packages/kit-bg/src/vaults/impls/neo/sdkNeo/signMessage.ts

```
@@ -91,15 +94,19 @@ export function isPublicKey(key: string, encoded?: boolean): boolean {
  * @returns Verification result, true for valid signature
  */
-export function verify(hex: string, sig: string, publicKey: string): boolean {
+export async function verify(
+  hex: string,
+  sig: string,
+  publicKey: string,
+): Promise<boolean> {
   try {
     if (!isPublicKey(publicKey, true)) {
       // eslint-disable-next-line no-param-reassign
       publicKey = getPublicKeyUnencoded(publicKey);
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)